ledger/participant-integration-api/src/main/scala/platform/store/dao/events/EventsTablePostgresql.scala

```diff
@@ -147,19 +148,58 @@ object EventsTablePostgresql extends EventsTable {
     )
   }
 
-  // Specific for PostgreSQL parallel unnesting insertions
```

Comment:
Strangely, Github doesn't let me comment on line 146 (does not appear changed), but as I see, `insertEvents` does not need to be an `Option`, since it's always constructed with `Some`.

Reply:
Fixed